src/js/_enqueues/wp/updates.js

```diff
@@ -2664,41 +2664,16 @@
 		} );
 
 		/**
-		 * Click handler for plugin activations in plugin activation view.
+		 * Click handler for plugin activations in plugin activation modal view.
 		 *
 		 * @since 6.5.0
+		 * @since 6.5.4 Redirect the parent window to the activation URL.
 		 *
 		 * @param {Event} event Event interface.
 		 */
-		$pluginFilter.on( 'click', '.activate-now', function( event ) {
-			var $activateButton = $( event.target );
-
+		$(document).on( 'click', '#plugin-information-footer .activate-now', function( event ) {
```

Comment:
```suggestion
		$document.on( 'click', '#plugin-information-footer .activate-now', function( event ) {
```

Reply:
@costdev small suggestion.

Reply:
Tested - just to make sure. Yes, works with no change in behavior 👍